firestore.rules

```diff
@@ -163,14 +163,6 @@ service cloud.firestore {
       allow read;
     }
 
-    // Note: `resource` = existing doc, `request.resource` = incoming doc
-    match /manalinks/{slug} {
-      // Only you can create a manalink with your fromId
-      allow create: if request.auth.uid == request.resource.data.fromId;
-      // Only you can list and change your own manalinks
-      allow list, update: if request.auth.uid == resource.data.fromId;
-    }
-
```

Comment:
Seems a little fishy to allow listing the old manalinks that used to be private which are still hanging around, unless you are also deleting all the old Firestore data (which would be reasonable, but seems a bit aggressive from a "what if I fucked up some part of the migration" perspective.)

Reply:
deleting the rule lets anyone access it?

Reply:
okay I removed the "cleanup" stuff from this